### PR TITLE
🔍  Added isJSONSchema helper method

### PR DESCRIPTION
**Pull request by nicolasgramlich**

### CODE REVIEW

File: src/utils/is-json-schema.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as Lodash from 'lodash';
+import { TJSONSchema } from '../types/json-schema';
+
+export function isJSONSchema(obj: {}): obj is TJSONSchema {
```
Comment (SylvainEstevez) [inline src/utils/is-json-schema.ts]: It's kinda tricky to check wether an object is a JSON schema or not, given that `{}` is a valid JSON schema. I wanted to add this helper originally but then thought I couldn't make it accurate.

Can you share the use case that led you to this addition?

**Comment by nicolasgramlich** [comment]
Something like `@body(oneOf([schemaA, schemaB])` will fail.

File: src/utils/is-json-schema.ts (hunk 1 above)
Comment (nicolasgramlich) [reply src/utils/is-json-schema.ts]: Something like `@body(oneOf([schemaA, schemaB])` will fail.

**Comment by nicolasgramlich** [comment]
🔖  `1.7.0`

**Pull request merged**